Makefile

```diff
@@ -63,6 +64,17 @@ start-FULL-REBUILD: echo_vars stop rm-ALL ## Remove and restart all Docker conta
 	docker-compose ${COMPOSE_FILE_ARGS} --env-file ${ENV_FILE} down
 	docker-compose ${COMPOSE_FILE_ARGS} --env-file ${ENV_FILE} up --build
 
+extract-bundles: ## Extract bundles from file-server for cloud deployment
+	/bin/rm -rf build
+	docker cp polis-${TAG}-file-server-1:/app/build/ build
```

Comment:
Will have to check that all of the build directories are the same now; Some of these changed with recent client PRs.